src/views/Home.vue

```diff
@@ -47,13 +48,21 @@
                 offName="star1"
                 @click="toggleFavorite(space.key)"
               />
-              <div class="">
+              <div>
                 <h3 v-text="space.name" />
                 <div class="text-gray">{{ space.symbol }}</div>
               </div>
             </Block>
           </div>
         </router-link>
+
+        <div
```

Comment:
You can use <Block> component here instead of <div> with the classes

Reply:
yes we can work on that separately later, here i havent done anything other than just removing the forgotten `class=""` text